db/primary_migrate/20240321130431_add_sign_in_new_device_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddSignInNewDeviceToUsers < ActiveRecord::Migration[7.1]
+  def change
+    add_column :users, :sign_in_new_device_at, :datetime
```

Comment:
Depending if we have any code that would be reading from this (see previous comment about preventing code from being run behind feature flag), we should consider if this needs to be migrated in a separate deploy:

https://handbook.login.gov/articles/manage-50-50-state.html#add-a-table-or-field

Reply:
Do you figure with the check for the feature flag in the job runner would it be OK to include with this now?

Reply:
The main thing is to be sure there's no code which would reference the column in a production environment. If all of the code that references the column is behind a feature flag, I think it'd be safe to keep in this pull request..